regression/exceptions/exceptions1/test.desc

```diff
@@ -3,8 +3,8 @@ test.class
 
 ^EXIT=10$
 ^SIGNAL=0$
-^.*assertion at file test.java line 24 function.*: FAILURE$
-\*\* 1 of 31 failed (2 iterations)$
+^.*assertion at file test.java line 26 function.*: FAILURE$
```

Comment:
If "Updated some regression tests for exceptions " is just fixing a previous commit in the same PR then please squash it